src/main/java/com/opinta/controller/ShipmentController.java

```diff
@@ -51,9 +51,10 @@ public ShipmentController(ShipmentService shipmentService, PDFGeneratorService p
         return new ResponseEntity<>(shipmentDto, OK);
     }
 
-    @GetMapping("{id}/label-form")
-    public ResponseEntity<?> getShipmentLabelForm(@PathVariable("id") long id) {
-        byte[] data = pdfGeneratorService.generateLabel(id);
+    @GetMapping("{id}/{parcelId}/label-form")
+    public ResponseEntity<?> getShipmentLabelForm(@PathVariable("id") long id,
+                                                  @PathVariable("parcelId") long parcelId) {
+        byte[] data = pdfGeneratorService.generateLabel(id, parcelId);
```

Comment:
because of changes of model. This pdf generator uses data both from parcel and shipment. So i decided to do it this way. Should i fix that?